chalicelib/responseobjects/hca_response_v5.py

```diff
@@ -298,13 +301,13 @@ def make_projects(self, entry):
             project_id = project["project"]
             translated_project = {
                 "shortname": project_id,
-                "laboratory": list(set(project.get("laboratory", None)))
+                "laboratory": list(set(project.get("laboratory")) if project.get("laboratory") else [])
```

Comment:
`"laboratory": list(set(project.get("laboratory", [])))`

Reply:
The problem is that project["laboratory"] can actually be None.